web/src/components/filings-calendar/tax-access/TaxAccessStepTwo.test.tsx

```diff
@@ -419,9 +440,8 @@ describe("<TaxAccessStepTwo />", () => {
 
     it("shows disclaimer text for TaxId", () => {
       renderComponent(userDataWithPrefilledFields);
-      expect(screen.getByText(Config.taxAccess.modalHeader)).toBeInTheDocument();
-
       const taxInput = screen.getByTestId("taxIdInput");
+      expect(within(taxInput).getByText("`New Jersey Tax ID|tax-id`")).toBeInTheDocument();
```

Comment:
dust - it feels odd this assertion is passing. the text in the DOM should read New Jersey Tax ID. Can you take a look as to how other contextual links are used in test? I'm sure there is a fn that strips the markdown syntax to identidy the contextual link.